python/mosaic/api/functions.py

```diff
@@ -510,6 +511,23 @@ def st_intersection(left_geom: ColumnOrName, right_geom: ColumnOrName) -> Column
         pyspark_to_java_column(right_geom),
     )
 
+def st_envelope(geom: ColumnOrName) -> Column:
+    """
+    Returns the minimum bounding box for the supplied geomtery.
```

Comment:
same as for the docs. @tdikland

Reply:
Yes, it's worth rephrasing "minimum bounding box" in "minimum rectangular bounding box"

Reply:
Oh, I see this was already addressed! Nice!